src/raft.rs

```diff
@@ -606,12 +649,74 @@ impl<T: Storage> Raft<T> {
     pub fn bcast_append(&mut self) {
         let self_id = self.id;
         let mut prs = self.take_prs();
-        prs.iter_mut()
-            .filter(|&(id, _)| *id != self_id)
-            .for_each(|(id, pr)| self.send_append(*id, pr));
+        if self.follower_delegate {
```

Comment:
Is it possible to not change `bcast_append`, but do all logics in `send_append`?

Reply:
I think both of these two approaches need to pick the delegate of each group first?

Reply:
I think `send_append` can be reused for this (maybe also need to do a little change for `send_append`). For many calls to `send_append` with a same delegate, we can just append a new `Commission` for it.

Reply:
I'm worried about too much state will be maintained in such an implementation.